### PR TITLE
Modulesync 0.12.7

### PR DESCRIPTION
**Pull request by bastelfreak**

### CODE REVIEW

File: .sync.yml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+---
+.travis.yml:
+	secure: "RrAmWtM6eTjZZzD954AIgR179Pqp14lzHhd/C9cpKbTPynLncuim08CEvjmq+7pgAy9XDg1d02x3udfZt4btR1sBdyNRpNN3yUhWptmGU61HRJdiZq+nSLQkIYsqXanhk+O3NndFojO58WRD01dkWEc6HRHjlivuYNxDXmMkg9k="
```
Comment (alexjfisher) [inline .sync.yml]: Sacre bleu!  A 'tab'?

File: manifests/password.pp [hunk 2]
```diff
@@ -40,18 +40,18 @@
...
-    content => $password_content,
     ensure  => present,
+    content => $password_content,
     require => Package[$package_name],
     notify  => Service[$virtual_service],
     tag     => "splunk_password",
   }
 
   file { "$secret_file":
+    ensure  => present,
```
Comment (alexjfisher) [inline manifests/password.pp]: Somehow, I thought this was a lint error in itself and we preferred `ensure => file`?

Comment (alexjfisher) [inline manifests/password.pp]: I guess we don't use https://github.com/voxpupuli/puppet-lint-file_ensure-check?

File: .sync.yml (hunk 1 above)
Comment (bastelfreak) [inline .sync.yml]: shitty vim paste mode :(

File: manifests/password.pp (hunk 2 above)
Comment (bastelfreak) [inline manifests/password.pp]: jup

**Pull request merged**